eng/common/testproxy/test-proxy-tool.yml

```diff
@@ -47,3 +47,18 @@ steps:
       displayName: "Run the testproxy - linux/mac"
       condition: and(succeeded(), ne(variables['Agent.OS'],'Windows_NT'))
       workingDirectory: "${{ parameters.rootFolder }}"
+
+    - pwsh: |
+        for ($i = 0; $i -lt 10; $i++) {
```

Comment:
We are in a great need to have a helper retry function for our PS scripts :)

Reply:
That we are!